CONTRIBUTING.adoc

```diff
@@ -0,0 +1,25 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/code-ready/crc/issues[Github issue tracker] for the repository.
+Before you submit a new issue, we recommend that you search the list of issues to see if anyone already submitted a similar issue.
+
+[[contributing-patches]]
+== Contributing patches
+
+Thank you for your contributions! Please follow this process to submit a patch:
+
+. Create an issue describing your proposed change to the repository.
+. The repository owners will triage and respond to your issue promptly.
+. Fork the repository and create a topic branch.
+. Submit a pull request with the proposed changes.
+.. The required format for the git commit message is 'Issue #<issue_no> message'.
+
+[[questions]]
+== Questions?
+
+If you run into issues or have any questions about contributions, feel free to ping the CRC developers on IRC at the #crc channel on Freenode.
```

Comment:
We don't have #crc channel as of now, are we planning to create it and move there instead of #code-ready one?

Reply:
@praveenkumar My mistake, it should be #codeready